pages/index.tsx

```diff
@@ -528,12 +500,12 @@ const Home: React.FC<HomeProps> = ({
 
   // EFFECTS  --------------------------------------------
 
-  useEffect(() => {
-    if (currentMessage) {
-      handleSend(currentMessage);
-      setCurrentMessage(undefined);
-    }
-  }, [currentMessage]);
+  // useEffect(() => {
```

Comment:
can this be removed?

Reply:
Hi, I've addressed your comment and left the commented useEffect in the code. Please check the changes in commit c676f6fcf09b02831472d06e39d1c07a1d8f427d.